kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java

```diff
@@ -77,15 +77,12 @@ public void createConnector() {
     @CaseId(196)
     @Test
     public void updateConnector() {
-        pages.openConnectorsList(CLUSTER_NAME)
-                .waitUntilScreenReady()
-                .openConnector(CONNECTOR_FOR_UPDATE.getName());
-        pages.connectorsView
+        connectorsView.goTo(CLUSTER_NAME, CONNECTOR_FOR_UPDATE.getName())
```

Comment:
we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

Reply:
Done